packages/core/src/app/common/utility/getAllowedPaymentMethodsForSubscription.ts

```diff
@@ -0,0 +1,9 @@
+import { PaymentMethod } from "@bigcommerce/checkout-sdk";
+
+export default function filterPaymentMethods(paymentMethods: PaymentMethod[], isSubscriptionEnabled: boolean | undefined): PaymentMethod[] {
+    if (!isSubscriptionEnabled) return paymentMethods;
+
+    return paymentMethods.filter(method =>
```

Comment:
@ptumash just so i can understand whats going on. we're only allowing credit cards, to pay for subscriptions? ie no paypal, etc